cmds/core/echo/echo.go

```diff
@@ -35,23 +56,20 @@ func escapeString(s string) (string, error) {
 	if err != nil {
 		return "", err
 	}
-
 	return s, nil
 }
 
-func echo(f flags, w io.Writer, s ...string) error {
+func echo(w io.Writer, noNewline, interpretEscapes bool, s []string) error {
```

Comment:
I think the variadic form (s ...string) should remain. the standard echo command takes zero arguments. It should be legal to call this function with no arguments.

I think the flags type should remain. That way, if we ever add new flags, we don't have to change the function signature. Not sure what motivates removing them.

Reply:
Regarding the variadic form I'm totally with you. Regarding the flags i think the most handy way is to access them directly by dereferencing, so no flag has to be passed. What do you think?

Reply:
looks good! Do you think our code coverage bot can be appeased?

Reply:
Jep, i did a minor change that appeased the coverage bot.